binary-install/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "binary-install"
```

Comment:
This crate will need to be published to crates.io to publish `wasm-pack` itself to crates.io, so could this perhaps be named `wasm-pack-binary-install` to namespace it?

Reply:
Sounds great, will rename it.